CONTRIBUTING.md

```diff
@@ -0,0 +1,183 @@
+# Contributing
+
+Firstly want to give a gigantic thank you for taking the time to contribute to this project. Your contribution is greatly appreciated.
+
+Before contributing, there is a few things you should know if you don't already.
+
+## Submitting a Pull Request
+
+### Dependencies
+
+In order to install these dependencies locally, you will need to simply run
```

Comment:
hmmg yes link to yarn maybe to instol

Reply:
https://github.com/eludris-community/evangeline/pull/9/commits/58bc0acfd2e41e500d49509109d2c163baf3e75a